lib/src/isolate_manager/isolate_manager_web.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:async';
+import 'dart:developer';
+
+import 'package:dash_kit_network/dash_kit_network.dart';
+import 'package:dash_kit_network/src/isolate_manager/isolate_manager.dart';
+
+// ignore: prefer-match-file-name
+class IsolateManager extends IsolateManagerInterface {
```

Comment:
let's add some docs that on web we cannot use isolates

Reply:
added.